demo/App.jsx

```diff
@@ -56,19 +61,35 @@ export default function App() {
               type="password"
               value={tokenInput}
             />
-            <Button color="blue" type="submit" onClick={handleSaveToken} disabled={!!token} ariaLabel="Save Token">
-              Save Token
-            </Button>
-            <Button type="button" onClick={handleClearToken} disabled={!token} ariaLabel="Clear Token">
-              Clear Token
-            </Button>
+            <span
+              style={{
+                display: 'flex',
+                alignItems: 'BaseLine',
+              }}
+            >
+              <Button color="blue" type="submit" onClick={handleSaveToken} disabled={!!token} ariaLabel="Save Token">
+                Save Token
+              </Button>
+              <Button type="button" onClick={handleClearToken} disabled={!token} ariaLabel="Clear Token">
+                Clear Token
+              </Button>
+              <Checkbox
```

Comment:
Not sure it makes sense to just add a random checkbox next to the buttons. You should create a separate `FedRAMP` section (before the `Access Token` section) that explains where to get a FedRAMP token and the checkbox to enable the environment